src/harness/sas_interface.py

```diff
@@ -256,6 +256,31 @@ def PreloadRegistrationData(self, request):
     """
     pass
 
+  @abc.abstractmethod
+  def InjectExZone(self, request):
+    """Inject PPA or NTIA zone information into SAS under test.
+
+    Args:
+      request: A dictionary with a single key-value pair where the key is
+        "record" and the value is ZoneData object to be injected into 
+        SAS under test. For more information about ZoneData please see 
+        the SAS-SAS TS (WINNF-16-S-0096).
+    """
+    pass
+
+
+  @abc.abstractmethod
+  def InjectZoneData(self, request):
```

Comment:
This is not needed - we already have a way to inject PPAs and we do not need to inject NTIA-defined zones.

Reply:
Already existing in master branch, changed